public/src/components/channelManagement/epicTests/epicTestVariantEditor.tsx

```diff
@@ -188,6 +193,14 @@ const EpicTestVariantEditor: React.FC<EpicTestVariantEditorProps> = ({
   const updateImage = (image?: Image): void => {
     onVariantChange({ ...variant, image });
   };
+  const updateBylineWithImage = (bylineWithImage?: BylineWithImage): void => {
+    if (bylineWithImage != null && bylineWithImage.headshot != null) {
```

Comment:
I think it would be better to have this logic in the `BylineWithImageEditor` component

Reply:
I can't think of a way of how to move the functionality into the component and get it to play nicely within the strictures imposed by `react-hook-form`. Maybe we/team can pair/mob on finding a better solution?

Reply:
Fixed